garage/src/pages/RigDetails/index.tsx

```diff
@@ -329,7 +327,7 @@ export const RigDetails = () => {
                   onOpenParkModal={onOpenParkModal}
                   onOpenPilotModal={onOpenPilotModal}
                   onOpenTrainModal={onOpenTrainModal}
-                  chainPilotStatus={pilotInfo?.status}
+                  chainPilotStatus={pilotInfo?.status === "success" ? 1 : 0}
```

Comment:
i'm realizing i made some incorrect assumptions, so this is wrong as well. the `wagmi` interface change now returns a `success` or `failure` value for a `status` key. with my changes, `pilotInfo?.status` was reading this value instead of something from the underlying `pilotInfo?.result`.

so this should maybe be `Number(pilotInfo?.result)`...but i'm still not sure how to get the correct value from the `wagmi` hook responses